tests/test_aligned_potential.py

```diff
@@ -0,0 +1,204 @@
+from jax import config
+
+config.update("jax_enable_x64", True)
+
+import jax.numpy as jnp
+import numpy as np
+from common import get_guest_params
+from numpy.typing import NDArray
+
+from timemachine.constants import DEFAULT_ATOM_MAPPING_KWARGS, NBParamIdx
+from timemachine.fe import atom_mapping
+from timemachine.fe.single_topology import AtomMapMixin, SingleTopology
+from timemachine.fe.topology import get_ligand_ixn_pots_params
+from timemachine.fe.utils import get_romol_conf, read_sdf_mols_by_name
+from timemachine.ff import Forcefield
+from timemachine.md.builders import build_protein_system
+from timemachine.potentials import BoundPotential, Nonbonded, NonbondedInteractionGroup
+from timemachine.utils import path_to_internal_file
+
+
+def _parameterize_host_nonbonded_reference(
```

Comment:
Again, not a new function, moved from `SingleTopology`

Reply:
(same comment as https://github.com/proteneer/timemachine/pull/1538/files#r2017057695)